suzieq/poller/worker/nodes/node.py

```diff
@@ -693,6 +706,7 @@ async def _init_ssh(self, init_dev_data=True, use_lock=True) -> None:
                     else:
                         self.logger.error('Unable to connect to '
                                           f'{self.address}:{self.port}, {e}')
+                        self._retry = 0
```

Comment:
I'm not sure we need it here. What if the device is temporarily unreachable, in that case we should keep trying.